lib/ansible/modules/get_url.py

```diff
@@ -487,7 +487,7 @@ def main():
         except ValueError:
             module.fail_json(msg="The checksum parameter has to be in format <algorithm>:<checksum>", **result)
 
-        if checksum.startswith('http://') or checksum.startswith('https://') or checksum.startswith('ftp://'):
+        if checksum.startswith('http://') or checksum.startswith('https://') or checksum.startswith('ftp://') or checksum.startswith('file://'):
```

Comment:
Maybe shortening this to
`if checksum.startswith(('http://', 'https://', 'ftp://', 'file://')):`
might be nice too.

Reply:
Yes, please use the suggestion above to shorten this statement.

Reply:
just making note for future, not holding this PR to it.

in the end we should really use parse_url and compare the 'schema' to 'supported' schema handlers, which startswith is 'kindof' doing here but it would be cleaner this way. Also would require to 'register' supported handlers and possibly giving a way to expand upon them.

Reply:
Made a function using urlsplit and using that. Possibility to expanding I'll leave to a later exercise.